toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt

```diff
@@ -61,6 +82,16 @@ public fun MapView(
     locationDisplay: LocationDisplay = rememberLocationDisplay(),
     mapViewInteractionOptions: MapViewInteractionOptions = MapViewInteractionOptionDefaults,
     onViewpointChanged: (() -> Unit)? = null,
+    isInteracting: ((Boolean) -> Unit)? = null,
```

Comment:
I suggest we name the parameter `onInteractionChanged` to be consistent with the other MapView related events, where each is prefixed with "on". Also suffixing it with "Changed" implies that there is a start and an end to map biew interaction, i.e. at start it passes `true` and at the end it passes `false`.
```suggestion
    onInteractionChanged: ((Boolean) -> Unit)? = null,
```

Reply:
>at start it passes true and at the end it passes false

I don't think that `onInteractionChanged` depicts accurately what it happening.  The api doc states 
`Indicates whether the user is currently interacting with the GeoView` 

With that in mind, in my opinion `onInteractionStarted` makes more sense. So then when the interaction starts it returns `true` and when it ends it returns `false`

and the doc should be changed from 
`@param isInteracting lambda invoked when the user is currently interacting with the composable MapView` 

to 
`@param onInteractionStarted lambda invoked when the user starts and ends the interaction with the composable MapView`